data/transactions/logic/eval_test.go

```diff
@@ -3910,15 +3917,28 @@ func obfuscate(program string) string {
 
 type evalTester func(pass bool, err error) bool
 
-func testEvaluation(t *testing.T, program string, introduced uint64, tester evalTester) error {
+func testEvaluation(t *testing.T, program string, introduced uint64, tester evalTester, xtras ...uint64) error {
```

Comment:
I needed to add `xtras` to re-gain the ECDSA tests which relied on `testEvaluationWithField()`

Reply:
I think I recall why it went away.  I think I made the error messages more uniform, so that the version test would be the  same for both. I don't think there's a great reason for one to be "... was introduced .." and the other to be "...available in version...".